<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/tray1.png</normaloff>images/tray1.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Manage accounts:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Add">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Edit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Delete">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mailcheck interval:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>86400</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Seconds</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Save">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sound file:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Sound">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Browse">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Play">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wait before first check:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Show notification for:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_Waitfirst">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>86400</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_Notificationtimeout">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>86400</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Seconds</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Seconds</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>7</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Lower right</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Upper right</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lower left</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Upper left</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Center</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Notification window:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>button_Add</tabstop>
  <tabstop>button_Edit</tabstop>
  <tabstop>button_Delete</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>line_Sound</tabstop>
  <tabstop>button_Browse</tabstop>
  <tabstop>button_Play</tabstop>
  <tabstop>button_Save</tabstop>
  <tabstop>button_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
